### ISSUE TITLE
Does not build on GCC-7

### ISSUE THREAD

**Issue by VincentSC** [GPUOpen-Tools/CodeXL#183]
It needs a `-Wno-implicit-fallthrough` to be able to buil sqlite.

**Comment by VincentSC** [comment]
GCC-6 gets further, but fails at miniz.c on its layout.
Then further on on ignored attributes..

Reason is -Werror, and gcc-version of 6.1 and later having a lot more warnings.

**Comment by VincentSC** [comment]
Work-around: remove `-Werror` from Common/Src/SCons/CXL_init.py

**Comment by pwasiewi** [comment]
Yes, after removing `-Werror` from all files, and adding some -Wno- , it stops at qt5 widget. Are there any given prerequisites requirements to compile and run CodeXL, I do it on Gentoo gcc73, kernel 4.15(no module), and its binary precompiled by you form crashes without any warning or trace and compilation progresses very slowly, there are still some bugs waiting for me, there.

**Comment by KingDuckZ** [comment]
+1 I have the same problem. Removing warning-related options seems to have fixed it so far (the build is still going). Either way, why is it bundled with sqlite? I'm pretty sure I have sqlite as well as qt already installed on my system, can't codexl just pick up the system libs instead of rebuilding everything from scratch?
Also, I think paralell build is not working. Which is not very helpful given the amount of code + libs that you have to compie...
